workspace/scm.py

```diff
@@ -187,19 +187,22 @@ def rename_branch(branch, new_branch):
     silent_run(['git', 'branch', '-m', branch, new_branch])
 
 
-def merge_branch(branch, commit=None, squash=False, strategy=None):
+def merge_branch(branch, commit=None, strategy=None, user=None):
     cmd = ['git', 'merge']
 
-    if squash:
-        cmd.append('--squash')
```

Comment:
Deprecating `squash` for now since the option is not supported by `wst merge` (also couldn't find any invocation of `merge_branch` with `squash` anywhere). We can implement support for `squash` option in the future if needed.